ims/einstein/ceph.py

```diff
@@ -186,6 +242,21 @@ def snap_unprotect(self, img_id, snap_name):
             raise file_system_exceptions.ImageBusyException(img_id)
 
     @log
+    def is_snap_protected(self, img_id, snap_name):
+        """
+        Find out whether a snapshot is protected from deletion
+
+        :param img_id: what the image is called
+        :param snap_name: the snapshot to check
+        :return: bool- whether the snapshot is protected
+        """
+        try:
+            with self.__open_image(img_id) as img:
+                return img.is_protected_snap(snap_name)
```

Comment:
I think, I got the answer from http://docs.ceph.com/docs/master/rbd/librbdpy/#rbd.Image.is_protected_snap, we will return our ImageNotFound exception. Correct me if I am wrong.

Also, where are we using this fn again? Is it in tests? Please see my comments below regarding fn's which are used by tests.